qr_generator/lib/utils/api_client.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:dio/dio.dart';
+
+class ApiClient {
+  late Dio dio;
```

Comment:
**fix**: with a factory constructor, this would be `final` and there wouldn't be the `late` overhead too (even if negligible)

Reply:
Done!